warehouse/cache/origin/fastly.py

```diff
@@ -143,3 +143,23 @@ def purge_key(self, key, metrics=None):
                     tags=[f"ip_address:{self.api_connect_via}"],
                 )
                 self._double_purge_key(key)  # Do not connect via on fallback
+
+
+@implementer(IOriginCache)
+class NullFastlyCache(FastlyCache):
+    """Same as FastlyCache, but it doesn't issue any requests"""
+
+    def _purge_key(self, key, connect_via=None):
+        path = "/service/{service_id}/purge/{key}".format(
+            service_id=self.service_id, key=key
+        )
+        url = urllib.parse.urljoin(self.api_endpoint, path)
+        headers = {
+            "Accept": "application/json",
+            "Fastly-Key": self.api_key,
+            "Fastly-Soft-Purge": "1",
+        }
+
+        print("Origin cache purge issued:")
+        print(f"* URL: {url!r}")
+        print(f"* Headers: {headers!r}")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏should we use print or logging?

Reply:
Given that we use `print` in other `Null`-services in local dev, I think this is probably fine, not sure if we'd get anything by using logging.